app/models/in_person_enrollment.rb

```diff
@@ -39,6 +39,27 @@ def needs_usps_status_check?(check_interval)
     )
   end
 
+  def complete?
+    return ['cancelled', 'expired', 'failed', 'passed'].include?(status)
+  end
+
+  def minutes_to_completion
+    return unless complete?
+    return unless enrollment_established_at.present?
+    return unless status_updated_at.present?
+    (status_updated_at - enrollment_established_at) / 60
+  end
+
+  def minutes_since_last_status_check
+    return unless status_check_attempted_at.present?
+    (Time.zone.now - status_check_attempted_at) / 60
+  end
+
+  def minutes_since_last_status_update
+    return unless status_updated_at.present?
+    (Time.zone.now - status_updated_at) / 60
```

Comment:
just an idea, maybe too DSL/railsy, but ActiveSupport has some helpers we could use
```suggestion
    (Time.zone.now - status_updated_at).seconds.in_minutes.floor
```

Reply:
Ended up adding `.round(2)` to limit it to 2 decimal places because long floats are kinda spammy